composer/trainer/trainer.py

```diff
@@ -1106,6 +1106,11 @@ def __init__(
         self._checkpoint_saver = None
         latest_remote_file_name = None
         if save_folder is not None:
+            if save_weights_only:
+                warnings.warn(
+                    'save_weights_only=True now also saves metadata and integrations! Please adjust your workflow accordingly.'
```

Comment:
Please specify a `stack_level` for warning.warn as is recommended (how far to unroll the traceback when displaying the warning)